Source/WebKit/WebProcess/WebPage/FindController.cpp

```diff
@@ -236,12 +230,24 @@ void FindController::updateFindUIAfterPageScroll(bool found, const String& strin
         }
         m_findPageOverlay->setNeedsDisplay();
     }
-    
+
     if (found && (!options.contains(FindOptions::ShowFindIndicator) || !selectedFrame || !updateFindIndicator(*selectedFrame, shouldShowOverlay)))
         hideFindIndicator();
+
+    completionHandler(idOfFrameContainingString, WTFMove(matchRects), matchCount, m_foundStringMatchIndex, didWrap == DidWrap::Yes);
 }
 
-void FindController::findString(const String& string, OptionSet<FindOptions> options, unsigned maxMatchCount, TriggerImageAnalysis canTriggerImageAnalysis, CompletionHandler<void(std::optional<FrameIdentifier>, bool)>&& completionHandler)
+#if ENABLE(IMAGE_ANALYSIS)
+void FindController::findStringInImages(const String& string, OptionSet<FindOptions> options, unsigned maxMatchCount, CompletionHandler<void(std::optional<FrameIdentifier>, Vector<IntRect>&&, uint32_t, int32_t, bool)>&& completionHandler)
```

Comment:
`findStringInImages` makes me think that it only finds matches in images. I think a better name for this function would indicate that it finds string matches in text and in images: maybe `findStringIncludingImages`